ingestion/src/metadata/orm_profiler/api/workflow.py

```diff
@@ -418,12 +438,16 @@ def execute(self):
 
         for database in databases:
             copied_service_config = self.copy_service_config(database)
-            sqa_metadata_obj = MetaData()
             try:
+                metadata_obj = MetaData()
+                if isinstance(copied_service_config, DatalakeConnection):
+                    metadata_obj = DataLakeProfilerType()
```

Comment:
You are passing the service config to the `self.create_profiler_interface` method, we should do the check there.

Reply:
resolved this